src/pages/Checkin/index.tsx

```diff
@@ -1,55 +1,77 @@
 import React, { useState } from 'react'
-import { Input, Title, Text, Button, PageTemplate } from '_/components'
-import { useAuth } from '_/contexts'
+import { Input, Title, Text, Button, PageTemplate, Form } from '_/components'
+import { useAuth, useCheckin } from '_/contexts'
 import carImg from "_/assets/CarApp.png"
 import { useNavigate } from 'react-router-dom'
 import { ROUTE_PATHS } from '_/constants'
 
 
 export function Checkin(){
     const [local, setLocal] = useState('')
+    const [duration, setDuration] = useState(1)
     const [errorMessage, setErrorMessage] = useState('')
     const { user } = useAuth()
     const navigate = useNavigate()
+    const { createCheckin, isCreatingCheckin } = useCheckin()
 
-    const handleOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const localOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
         setLocal(e.target.value)
         setErrorMessage('')
     }
 
-    const send = () => {
+    const durationOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
how about "onDurationChange" as the name of this function?

Reply:
done